ui/src/app/machines/views/MachineList/MachineList.js

```diff
@@ -60,6 +60,14 @@ const getSortValue = (machine, currentSort) => {
   }
 };
 
+const pluraliseMachineCount = count => {
```

Comment:
I reckon we should just use pluralize and do away with having hundreds of little pluralizing functions everywhere: https://www.npmjs.com/package/pluralize
It's obviously heavier than ternaries, but... it's just neater.

Reply:
Good call. Done